config/supported_ssg.yaml

```diff
@@ -4,6 +4,7 @@ supported:
   RHEL-6.6:
     package: scap-security-guide-0.1.18-3.el6
     version: 0.1.18
+    upstream_version: N/A
```

Comment:
There's really no need to add these if we just assume `nil` means the upstream_version is the same as the downstream_version.

Reply:
The problem is that `nil` is means `version` (when upstream and downstream version matches). We'll get rid of this once we start downloading downstream pkgs.

Reply:
Sorry, I don't follow. `nil` would mean the upstream and downstream versions are the same, like you said.

Reply:
I guess there is a misunderstanding. What was your initial concern around using "N/A", which means there is no upstream package (ZIP)?

Reply:
Since it's not part of the original config given to us by OpenSCAP, we could have just assumed a `nil` or missing upstream_version meant that the downstream and upstream version were the same. Using 'N/A', we have to go add that to the config every time it's updated (every RHEL release). There's really no reason to use a special string ('N/A') instead of nil/missing value, but there are good reasons against it - mostly because it simplifies the code but also because we only have to modify the records that have different upstream versions rather than all the records.

Reply:
The `upstream_version` is defined only when it differs from downstream version. The `upstream_version` was set by us in this PR. If it is not defined (missing) or set to nil the subsequent code will pick value `version`. Note, that not defined is the same as `nil`, because of a default value in the `Struct`. The "N/A" has a different purpose.

I agree that it is not very ideal. We could set the `upstream_version` everywhere, and proclaim `nil` as the new "N/A". I don't have more information on the upstream vs. downstream versions, other than on those few defined here.

Reply:
> The "N/A" has a different purpose.
Oh, what does 'N/A' mean? That the version doesn't exist at all upstream?

Reply:
Yes, that the version does not have a ZIP available to download from upstream.
Based on your suggestion from https://github.com/RedHatInsights/compliance-backend/pull/600#discussion_r504970115

What did I miss? :smile:

Reply:
Ok, makes sense. Sorry for the confusion! :)